src/agent/CMakeLists.txt

```diff
@@ -28,7 +29,7 @@ set(SOURCES
 
 add_library(Agent ${SOURCES})
 target_include_directories(Agent PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
-target_link_libraries(Agent PUBLIC ConfigurationParser Communicator AgentInfo MultiTypeQueue PRIVATE OpenSSL::SSL OpenSSL::Crypto Boost::asio Boost::beast)
+target_link_libraries(Agent PUBLIC ConfigurationParser Communicator AgentInfo MultiTypeQueue SQLiteManager PRIVATE OpenSSL::SSL OpenSSL::Crypto Boost::asio Boost::beast)
```

Comment:
At this point in time (in the git history) is not necessary to link to the SQLiteManager. This could be removed.

Reply:
Done